src/components/Search/Styles.tsx

```diff
@@ -0,0 +1,242 @@
+import React from "react"
+import { Global, css } from "@emotion/react"
+
+const styles = css`
```

Comment:
As this is a component
```suggestion
const Styles = css`
```

Reply:
Oh its not...my bad. Ignore this.

Reply:
:-) Gonna refactor using approach here anyway:
https://github.com/chakra-ui/chakra-ui-docs/blob/main/src/components/search.styles.tsx

That will remove this variable, and allow us to replace all the css `var()` instances with JS.

Reply:
Per our discussion, pushed an update to use returned `theme` variable, but did not use this for colors as it created issues with color mode responsive semantic tokens.